app/controllers/ui_ansible_roles_controller.rb

```diff
@@ -7,6 +10,12 @@ def index
     @ui_ansible_roles = resource_scope_for_index(:permission => :view_ansible_roles)
   end
 
+  api :DELETE, '/ui_ansible_roles/:id', N_('Deletes Ansible role')
+  param :id, :identifier, :required => true
+  def destroy
+    AnsibleRole.find_by(id: params[:id]).destroy
```

Comment:
Fixed, but the safe navigation operator could not be used, because then a 2XX code is returned, which leads to 
"\<role_name\> was successfully deleted" being shown in the UI.